packages/core/src/utils.ts

```diff
@@ -555,7 +555,7 @@ export function deepClone<T>(input: T): T {
  * @param input The input string.
  * @returns True if the input string matches the UUID format.
  */
-export function isUUID(input: string): boolean {
+export function isUUID(input: string): input is `${string}-${string}-${string}-${string}-${string}` {
```

Comment:
Curious if this helps at a particular call site?

Reply:
Yeah, I remember something in the request ID middleware needed it